modules/openapi-generator/src/main/resources/go/api.mustache

```diff
@@ -4,17 +4,17 @@ package {{packageName}}
 {{#operations}}
 import (
 	"bytes"
-	_context "context"
-	_ioutil "io/ioutil"
-	_nethttp "net/http"
-	_neturl "net/url"
+	"context"
```

Comment:
What if one of the parameter names in the operation is `context`? Will that result in issues after this change?

Reply:
iirc all methods used are upper-case as they would not be exported otherwise, meaning that `context` can not conflict with one of those. However, I might have missed something - do you have a specific place in mind that could conflict here?

Reply:
Ok, so I looked into this. The thing is, `context` is only used in a few places - primarily in structs, variable declarations, and function signatures:

```
type AdminApiApiCreateIdentityRequest struct {
	ctx            context.Context
	ApiService     *AdminApiService
	createIdentity *CreateIdentity
}
var (
	_ context.Context
)

func (a *AdminApiService) CreateIdentity(ctx context.Context) AdminApiApiCreateIdentityRequest {
	return AdminApiApiCreateIdentityRequest{
		ApiService: a,
		ctx:        ctx,
	}
}
```

In all of those cases there can never be an overshadowing variable clash. So yes, this is safe!